csrc/custom_all_reduce.cuh

```diff
@@ -23,17 +24,25 @@
 
 namespace vllm {
 
-constexpr int kMaxBlocks = 64;
-// note: we don't want to use atomics for signals because peer atomics are no
-// supported on PCIe links
+constexpr int kMaxBlocks = 36;
```

Comment:
why change kMaxBlocks 64 to 36?  I think the max block nums is related to topology bandwidth, I'm curious about the reasons for changing it.

Reply:
Currently, the number of blocks is hardcoded to 36, so there's no reason to set `kMaxBlocks` to a number larger than that.